tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -113,6 +113,21 @@ public function testAlternativeNames(): void
                         'type' => 'App\Foo',
                         'groups' => ['group1', ['group2', 'parent' => 'child3']],
                     ],
+                    [
+                        'alias' => 'Foo1',
+                        'type' => 'App\Foo',
+                        'options' => null,
+                    ],
+                    [
+                        'alias' => 'Foo1',
+                        'type' => 'App\Foo',
+                        'options' => ['foo' => 'bar'],
+                    ],
+                    [
+                        'alias' => 'Foo1',
+                        'type' => 'App\Foo',
+                        'serializationContext' => ['useJms' => false],
```

Comment:
Lets also test that anything can be used here

```suggestion
                        'serializationContext' => ['useJms' => false, 'foo' => 'bar'],
```